### ISSUE TITLE
<Leader>b gets weirdly filtered with let g:Lf_RememberLastSearch = 1

### ISSUE THREAD

**Issue by MightyCreak** [Yggdroot/LeaderF#89]
With `let g:Lf_RememberLastSearch = 0` I can simply switch to the previous buffer by doing `<Leader>b<Enter>`, but when I do `let g:Lf_RememberLastSearch = 1` then the same command put me back on the same buffer.

Workaround: `<Leader>b<Backspace><Enter>`

**Comment by Yggdroot** [comment]
It works as designed. If you just want to switch to previous buffer, you don't have to use LeaderF, use `ctrl-6`.

**Comment by MightyCreak** [comment]
Thanks for Ctrl-6, didn't know this one.

Though I really think the described behaviour should not be the default design.

Say we have 4 buffers (bold is the current buffer, italic is the previous one): **aaaa**, aabb, bbbb, _bbaa_

With RememberLastSearch _disabled_, if I hit <Leader>B I got this list:
- _bbaa_
- aabb
- bbbb
- **aaaa**

With RememberLastSearch _enabled_, if I hit <Leader>B I got this list:
- **aaaa**
- aabb
- bbbb
- _bbaa_

Although I have search for nothing before. Don't you find it weird that the behaviour is different in this case? When I enable RememberLastSearch, I expect to keep the same search words, but not to get the same search results (at least, that's how I pictured it in my mind when I proposed it to you ;) ).

I don't really care to remember the last search results, since the asynchronous system allows to retrieve very quickly the filtered results based on the remembered search words.
